uberpoet/filegen.py

```diff
@@ -88,6 +110,46 @@
 """
 
 
+def get_func_call_template(from_language, to_language, function_type):
+    if function_type == FuncType.SWIFT_ONLY:
+        if from_language == Language.OBJC:
+            raise ValueError("Cannot invoke SWIFT_ONLY method from ObjC!")
+        return swift_to_swift_func_call_template if to_language == Language.SWIFT else swift_to_objc_func_call_template
+    elif function_type == FuncType.OBJC_FRIENDLY:
+        if from_language == Language.SWIFT:
```

Comment:
in python I think you can compare tuples, so you could do `(from_lang, to_lang) == (Language.SWIFT, Language.OBJC)` and not have nested if statements